packages/react/src/transactions/adapters/useUpdateFollowPolicyController.ts

```diff
@@ -8,37 +8,48 @@ import {
   UpdateFollowPolicy,
   UpdateFollowPolicyRequest,
 } from '@lens-protocol/domain/use-cases/profile';
-import { BroadcastingError } from '@lens-protocol/domain/use-cases/transactions';
+import { BroadcastingError, SubsidizeOnChain } from '@lens-protocol/domain/use-cases/transactions';
 import { PromiseResult } from '@lens-protocol/shared-kernel';
 
 import { useSharedDependencies } from '../../shared';
 import { TransactionResultPresenter } from './TransactionResultPresenter';
-import { UpdateFollowPolicyCallGateway } from './UpdateFollowPolicyCallGateway';
+import { UpdateFollowPolicyGateway } from './profiles/UpdateFollowPolicyGateway';
 import { validateUpdateFollowPolicyRequest } from './schemas/validators';
 
 export function useUpdateFollowPolicyController() {
-  const { activeWallet, apolloClient, onChainRelayer, transactionGateway, transactionQueue } =
-    useSharedDependencies();
+  const {
+    activeWallet,
+    apolloClient,
+    onChainRelayer,
+    transactionFactory,
+    transactionGateway,
+    transactionQueue,
+  } = useSharedDependencies();
 
   return async (
     request: UpdateFollowPolicyRequest,
   ): PromiseResult<
     void,
     | BroadcastingError
     | PendingSigningRequestError
+    | TransactionError
```

Comment:
I don't think the `TransactionError` are returned here... Only in the `AsyncTransactionResult<T>`. Unless we didn't use that pattern here.

Reply:
Worth investigating if we used `AsyncTransactionResult<T>` at all in any of these: follow, unfollow, and update profile metadata.

Reply:
I see they all use `TransactionResultPresenter` so they do.

Might just be an error in the type def then.

Reply:
I don't know what to do about it then